src/encoded/static/components/schema.js

```diff
@@ -629,7 +629,8 @@ globals.contentViews.register(SchemaPage, 'JSONSchema');
 // add a new object of that type if you're logged in.
 const AllSchemasPage = (props, reactContext) => {
     const { context } = props;
-    const loggedIn = !!(reactContext.session && reactContext.session['auth.userid']);
+    const reactContextUser = reactContext.session_properties ? reactContext.session_properties.user : {};
+    const hasLab = !!(reactContextUser && reactContextUser.lab && reactContextUser.lab.status === 'current');
```

Comment:
I think technically `reactContextUser` will always be truthy, because if session_properties is nothing, then you set reactContextUser to an empty object, which is truthy.

Reply:
Switching between Python and Javascript... Python evaluate to the key-value nature of object-{} has the opposite effect. I will fix this.